lms/djangoapps/grades/new/course_grade.py

```diff
@@ -24,8 +27,40 @@ class CourseGrade(object):
     def __init__(self, student, course, course_structure):
         self.student = student
         self.course = course
+        self.course_version = getattr(course, 'course_version', None)
+        self.course_edited_timestamp = getattr(course, 'subtree_edited_on', None)
```

Comment:
My assumption was that this was done to save having to call `getattr` when we want to access these attributes. That said, we're only using them when we create the grade in the database so it's not like there are currently multiple call sites. That will likely change when we add events.